src/brogue/Items.c

```diff
@@ -4318,10 +4396,14 @@ boolean updateBolt(bolt *theBolt, creature *caster, short x, short y,
                 }
                 break;
             case BE_NEGATION:
-                negate(monst);
+                negated = negate(monst);
                 if (boltCatalog[BOLT_NEGATION].backColor) {
                     flashMonster(monst, boltCatalog[BOLT_NEGATION].backColor, 100);
                 }
+                if (rogue.patchVersion >= 4 && negated && autoID) {
```

Comment:
Changes look great, thanks - was *just* about to merge, then I remembered it needed this:

```suggestion
                if (rogue.patchVersion >= 4 && negated && autoID && canSeeMonster(monst)) {
```